src/python/tests/appengine/handlers/jobs_test.py

```diff
@@ -54,54 +57,20 @@ def _create_job(self,
 
   def test_get_results(self):
     """Test get_results."""
-    job = self._create_job('test_job', 'APP_NAME = launcher.py\n')
-    expected = {
-        'templates': [],
-        'jobs': [job],
-        'fieldValues': {
-            'csrf_token':
-                None,
-            'queues': [{
-                'display_name': 'Android',
-                'name': 'ANDROID'
-            }, {
-                'display_name': 'Android (x86)',
-                'name': 'ANDROID_X86'
-            }, {
-                'display_name': 'Android Kernel',
-                'name': 'ANDROID_KERNEL'
-            }, {
-                'display_name': 'Chrome OS',
-                'name': 'CHROMEOS'
-            }, {
-                'display_name': 'Fuchsia OS',
-                'name': 'FUCHSIA'
-            }, {
-                'display_name': 'Linux',
-                'name': 'LINUX'
-            }, {
-                'display_name': 'Linux (untrusted)',
-                'name': 'LINUX_UNTRUSTED'
-            }, {
-                'display_name': 'Linux (with GPU)',
-                'name': 'LINUX_WITH_GPU'
-            }, {
-                'display_name': 'Mac',
-                'name': 'MAC'
-            }, {
-                'display_name': 'Windows',
-                'name': 'WINDOWS'
-            }, {
-                'display_name': 'Windows (with GPU)',
-                'name': 'WINDOWS_WITH_GPU'
-            }],
-            'update_job_template_url':
-                '/update-job-template',
-            'update_job_url':
-                '/update-job',
-            'upload_info':
-                collections.OrderedDict(),
-        },
-    }
-    results = jobs.Handler.get_results()
-    self.assertEqual(expected, results)
+    self.mock.has_access.return_value = True
+    job1 = self._create_job('test_job1', 'APP_NAME = launcher1.py\n')
+    job2 = self._create_job('test_job2', 'APP_NAME = launcher2.py\n')
+    job3 = self._create_job('test_job3', 'APP_NAME = launcher3.py\n')
+    job4 = self._create_job('test_job4', 'APP_NAME = launcher4.py\n')
+
+    expected_items = [
+        job1.key.id(),
+        job2.key.id(),
+        job3.key.id(),
+        job4.key.id()
+    ]
+
+    resp = self.app.post_json('/', {'page': 1})
```

Comment:
Can you add a test for the searching part too?

Reply:
Should I make a separate unit-test for searching or add it in the same?

Reply:
let's make a separate one.